### DISCUSSION TITLE
Solved: How to see the menu, as described in the documentation?

### DISCUSSION THREAD

**Discussion by udippel** [gerbera/gerbera#2850] [General]
Yes, I manage to access the UI, with username and password. I have enabled tooltips, account. 
Though there is nothing like what is shown on the images, no "wombat", no "Welcome Menu". Immediately and only the database/filesystem page. So I can't change the config from here; can't see the clients ... .

How can I get to the page as depicted in the documentation, please?

**Comment by udippel** [comment]
So, this is my start page, directly after login.
![Screenshot_20230809_202307](https://github.com/gerbera/gerbera/assets/50265752/c310ada1-fb0c-4f4b-b020-11ea2fc30ba8)

**Reply by KarlStraussberger** [reply]
This is a very old version of gerbera.

What is your system, where did you get gerbera from.

**Reply by udippel** [reply]
Easy. Ubuntu 22.04. Stock. Daily updates.
`ii  gerbera     1.1.0+dfsg-3ubuntu4   amd64        UPnP MediaServer`
"We strongly advise using the packages provided by your distribution if available."

What do you suggest?

**Reply by KarlStraussberger** [reply]
You should try to get the package from jfrog https://gerbera.jfrog.io/ui/packages

**Reply by udippel** [reply]
Will try, and come back only in case things don't work out, unexpectedly.

**Reply by udippel** [reply]
Okay, found one file gerbera_1.11.0-jammy1_amd64.deb in Artefacts. Hope this is the correct one? Search had not delivered anything for "ubuntu" nor "Jammy".
Now the UI looks similar to the description. However, it seems autoscan doesn't work. I have selected a directory of media files, and now it shows 
`warning: Web Error: 800 Error: [/build/src/content/autoscan.cc:248] static AutoscanDirectory::ScanMode AutoscanDirectory::remapScanmode(const string&) Error: Illegal scanmode () given to remapScanmode()
`
and doesn't do anything.

What am I doing wrong?

**Reply by udippel** [reply]
I think I can answer my own question. It is a bit counter-intuitive, but at first one has to "Add [an] Item", so the folder gets scanned. Once this is done, it is possible to "Add Autoscan Item".

**Reply by KarlStraussberger** [reply]
If you are in the Filesystem view you can also add a complete folder to the autoscan list or you can add the folder the `config.xml` autoscan section (https://docs.gerbera.io/en/latest/config-import.html#autoscan).
